src/RegionLabel/index.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import React, { useState, memo } from "react"
+import React, { useRef, memo } from "react"
 import Paper from "@material-ui/core/Paper"
 import { makeStyles } from "@material-ui/core/styles"
 import styles from "./styles"
```

Comment:
you need to import TextField (it doesn't compile?)

Reply:
@seveibar 
That's right, sorry I missed that